src/pocapi.cpp

```diff
@@ -62,7 +76,9 @@ OVMS_Status* OVMS_ServerNew(OVMS_Server** server) {
 }
 
 OVMS_Status* OVMS_ServerDelete(OVMS_Server* server) {
-    // Make use of the server pointer instead of singleton once multi server configuration becomes possible.
+    if (server == nullptr) {
```

Comment:
Why is the comment removed?

Reply:
Because I sam that we already use the server from argument. Moved commend at the end of function since we cannot delete this server.